lightwood/api/data_source.py

```diff
@@ -209,8 +213,18 @@ def get_column_original_data(self, column_name):
         return self.data_frame[column_name].tolist()
 
     def lookup_encoder_class(self, column_type):
-        encoder_class = importlib.import_module(f'lightwood.encoders.{column_type}').default
-        return encoder_class
+        default_encoder_classes = {
```

Comment:
according to the guidelines that we've agreed on, I think it should be an `if/elif` statement rather than a `dict` lookup